source/codegen/templates/service_helpers.mako

```diff
@@ -187,6 +187,8 @@ one_of_case_prefix = f'{namespace_prefix}{function_name}Request::{PascalFieldNam
 % elif c_type == 'ViSession':
       auto ${parameter_name}_grpc_session = ${request_snippet};
       ${c_type} ${parameter_name} = session_repository_->access_session(${parameter_name}_grpc_session.id(), ${parameter_name}_grpc_session.name());\
+% elif c_type == 'ViUInt32[]' or c_type == 'ViInt32[]':
```

Comment:
I don't think a `const_cast` of a `reinterpret_cast` is the right implementation. I thought we already supported `ViInt32[]` and `ViUInt32[]`. The `reinterpret_cast` can easily hide bugs. What's the error you're seeing with the `const_cast`?

Reply:
@jonlugoatx I am getting this error when using `const_cast`
![image](https://user-images.githubusercontent.com/32991453/117918594-f23b1280-b308-11eb-8a44-725ab4a9931d.png)

Reply:
I have removed the changes and the APIs requiring it. they will be added in a separate PR

Reply:
The important part of the error is what comes after the "with". A `const google::protobuf::uint32*` which is really a `const uint32_t*` should be convertible to a `ViUInt32*` using a `const_cast`. Are we sure the proto file is using `repeated uint32` for `ViUInt32[]` types?

Reply:
I will give that a look and follow up in the PR adding those APIs